lib/livebook_web/live/session_live/app_docker_component.ex

```diff
@@ -80,13 +89,42 @@ defmodule LivebookWeb.SessionLive.AppDockerComponent do
         You can deploy this app in the cloud using Docker. To do that, configure
         the deployment and then use the generated Dockerfile.
       </p>
-      <p class="text-gray-700">
-        <.label>Hub</.label>
-        <span>
-          <span class="text-lg"><%= @hub.hub_emoji %></span>
-          <span><%= @hub.hub_name %></span>
-        </span>
-      </p>
+      <div class="flex gap-12">
+        <p class="text-gray-700">
+          <.label>Hub</.label>
+          <span>
+            <span class="text-lg"><%= @hub.hub_emoji %></span>
+            <span><%= @hub.hub_name %></span>
+          </span>
+        </p>
+        <%= if @hub_type == "team" do %>
+          <%= if @deployment_groups != [] do %>
+            <.form
+              :let={f}
+              for={@deployment_group_form}
+              phx-change="select_deployment_group"
+              phx-target={@myself}
+              id="select_deployment_group_form"
+            >
+              <.select_field
+                help={
+                  ~S'''
+                  Deployment group.
```

Comment:
Share deployment credentials, secrets, and configuration with deployment groups.